Building-with-IntelliJ-IDEA.md

```diff
@@ -52,64 +50,97 @@ When first viewing the project in IntelliJ IDEA you may come across this error:
 
-If you are having issues with missing JDK see [Troubleshooting](#troubleshooting) section.
-You can now save this build step as *Run configuration*, just right click *Install* again and select *Create..* (*Modify Run Configuration...* in IntelliJ 2021.2), window like this should pop-up, so just hit *Ok* (you can append `-DskipTests` for faster builds):
+At this point, it should start the build, which might take a while. Once the process is done, you should see a message to let you know that the build was successful:
 
-![project-maven-install-save](https://i.imgur.com/vdJoJ7L.png)
+![successful_build.png](img%2Fbuilding-with-intellij%2Fsuccessful_build.png)
 
-Now, we need to tell IntelliJ to pick up changes based on Maven build (it should do that automatically, but sometimes it doesn't). So, still in same Maven window, just click the Refresh icon:
+We now need to tell IntelliJ to pick up changes based on Maven build (it should do that automatically, but sometimes it doesn't). `Generate Sources and Update folders for all projects`
 
-![project-maven-refresh](https://i.imgur.com/L2SmWtF.png)
+![generate_sources.png](img%2Fbuilding-with-intellij%2Fgenerate_sources.png)
 
 ## Running the project
 
-Now locate *Project* in sidebar and click that open project view, and expand the tree to `runelite/runelite-client/src/main/java/net/runelite/client`. And all you need to do now is right-click `RuneLite` class and select run, and you are done, RuneLite should open:
+Now locate `Project` in sidebar and click that open project view, and expand the tree to `runelite/runelite-client/src/main/java/net/runelite/client`. 
+And all you need to do now is right-click the `RuneLite` class and select `run ‘RuneLite.main()’`
+
+![run_runelite.png](img%2Fbuilding-with-intellij%2Frun_runelite.png)
 
-![project-view-run-main](https://i.imgur.com/RXkMc48.png)
+And you are done! RuneLite should open.
 
 If you are having any issues with this step see [Troubleshooting](#troubleshooting) section.
 
 ## Conclusion
 
-Success! You can switch between running the client and building the Maven project by switching between the **RuneLite** configuration and your custom Maven configuration (should be named *runelite-parent [install]* by default):
+Success! You can now rebuild RuneLite by pressing the green play button near the top right corner of IntelliJ.
 
-![select-run-config](https://i.imgur.com/zKSqojU.png)
+![quick_run_runelite.png](img%2Fbuilding-with-intellij%2Fquick_run_runelite.png)
 
 # Troubleshooting
 
 ## Missing git
 
 If you are getting error about `git.exe` (or `git` on linux and mac) missing, you will need to first download and install Git for your OS. Git is version control software and implementation that RuneLite uses to store and track history of it's source code. To download git, just [go here and select your OS version](https://git-scm.com/downloads).
 
-## Missing JDK
+## Building on master branch fails
```

Comment:
I've told you four times now that the existing instructions for this work. You just started trying random things instead of asking for help with your error, and now you want to add these random things to the build guide.

Reply:
And I have told you countless times that it's not detailed enough. I have tried following the existing instructions multiple times, and never got it to work. And I'm not the only one who had this issue.  It took a long while watching random videos on youtube and testing things out to get it working.

Reply:
> You just started trying random things instead of asking for help with your error, and now you want to add these random things to the build guide.

Reply:
If you had asked for help, there's a section of the troubleshooting section I would have pointed you to, that I'm almost certain would have fixed your errors

Reply:
To reiterate previous conversations from discord, your setup was/is *wrong* and rewriting the guides so it works for your scenario isn't correct. Anything mentioning tags should be removed. 

We are mostly interested in **updated** screenshots/build steps that might have changed between intellij version updates. Ala VM options, maven icons, etc.

Reply:
> And doing this additional step fixed my issue. I don't see what's your problem with this?

My problem is you are trying to incorrectly instruct future users on how to resolve this issue, and teaching them an incorrect standard for opensource contributions, due to the fact that you didn't want to use the troubleshooting section or ask for help, to resolve your build errors.

Reply:
> > And doing this additional step fixed my issue. I don't see what's your problem with this?
> 
> My problem is you are trying to incorrectly instruct future users on how to resolve this issue, and teaching them an incorrect standard for opensource contributions, due to the fact that you didn't want to use the troubleshooting section or ask for help, to resolve your build errors.

The guide I made was working with the checkout step. didn't work without it. Everything else seems to be similar in terms to the current guide?

Building from master sometimes fails, so I don't see anything wrong with telling users to try checking out to a stable branch to make sure the IDE is set up correctly in the troubleshooting section, to rule out everything else.

Reply:
If building from master is failing, you've done something wrong. Providing a workaround for a situation that arises when the guide was not followed correctly is unhelpful.

Reply:
This is going to be my last comment on this situation ***and*** pull request.

I'm not entirely sure of your history with working in opensource and collaborative projects but when 3 different maintainers ask you to not do something, you shouldn't do it. It's been noted ***by the maintainers*** that your addition is misleading/wrong and how your setup was wrong in the first place. We've been more than welcoming to your other changes but if you aren't willing to concede on this opinion, then we can just close this and move on.